public/js/SettingIconLastBuyPriceRemoveThreshold.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable no-undef */
+class SettingIconLastBuyPriceRemoveThreshold extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      lastBuyPriceRemoveThresholds: {}
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  componentDidUpdate(nextProps) {
+    // Only update configuration, when the modal is closed and different.
+    if (
+      _.isEmpty(nextProps.lastBuyPriceRemoveThresholds) === false &&
+      _.isEqual(nextProps.lastBuyPriceRemoveThresholds, this.state.lastBuyPriceRemoveThresholds) ===
+      false
+    ) {
+      const { lastBuyPriceRemoveThresholds } = nextProps;
+      console.log('lastBuyPriceRemoveThreshold has changed', { lastBuyPriceRemoveThresholds });
+      this.setState({
+        lastBuyPriceRemoveThresholds
+      });
+    }
+  }
+
+  handleInputChange(event) {
+    const target = event.target;
+    const value =
+      target.type === 'checkbox'
+        ? target.checked
+        : target.type === 'number'
+          ? +target.value
+          : target.value;
+    const stateKey = target.getAttribute('data-state-key');
+
+    const { lastBuyPriceRemoveThresholds } = this.state;
+
+    console.log(
+      '_.set(lastBuyPriceRemoveThresholds, stateKey, value) => ',
+      _.set(lastBuyPriceRemoveThresholds, stateKey, value)
+    );
+
+    const newLastBuyPriceRemoveThresholds = _.set(lastBuyPriceRemoveThresholds, stateKey, value);
+    this.setState({
+      lastBuyPriceRemoveThresholds: newLastBuyPriceRemoveThresholds
+    });
+
+    this.props.handleLastBuyPriceRemoveThresholdChange(lastBuyPriceRemoveThresholds);
+  }
+
+  render() {
+    const { quoteAssets } = this.props;
+    const { lastBuyPriceRemoveThresholds } = this.state;
+
+    if (_.isEmpty(lastBuyPriceRemoveThresholds)) {
+      return '';
+    }
+
+    return quoteAssets.map((quoteAsset, index) => {
+      return (
+        <div
+          key={quoteAsset + '-' + index}
+          className='coin-info-last-buy-remove-threshold-wrapper'>
+          <Form.Group
+            controlId={'field-min-last-buy-remove-threshold-limit-percentage-' + quoteAsset}
+            className='mb-2'>
+            <Form.Label className='mb-0'>
+              Last Buy Price Remove Threshold for {quoteAsset}{' '}
```

Comment:
As mentioned earlier, the global setting screen does not match with the symbol setting screen.

- The symbol setting screen has `Remove last buy price when under` but in the global setting screen has `Last Buy Price Remove Threshold for USDT`. I think `Remove last buy price when under` is easier to understand. So let's go with it.
- One more suggestion is let's display the field with the quote asset like the below screenshot. |     10 | USDT |
   ![image](https://user-images.githubusercontent.com/5715919/121688054-60d8df00-cb06-11eb-987a-4ec54beba2e4.png)


![image](https://user-images.githubusercontent.com/5715919/121687797-19525300-cb06-11eb-8bce-bcea4f518eb7.png)

Reply:
Sure! Can you send me the code to do this? Don't forget I'm newbie in js.